osu.Game/Graphics/UserInterface/OsuSliderBar.cs

```diff
@@ -126,7 +127,6 @@ protected override void OnUserChange(T value)
         {
             base.OnUserChange(value);
             playSample(value);
-            updateTooltipText(value);
```

Comment:
This change regresses tooltip update on `TransferValueOnCommit` case.

Reply:
Yep, this needs to remain